### PR TITLE
Propose requiring use of the policy config system for some features

### PR DESCRIPTION
**Pull request by optimumtact**
This commit proposes adding the requirement that a developer working on
a ghost role, antag, minor antag or event mob role of any kind must
support the policy configuration system. It also lays out a simple guide
to that system, as well as expectations.

The requirement will go into affect upon merge, any non merged PR at
that time will be required to implement the guidelines

### CODE REVIEW

**Review by Cyberboss — DISMISSED** [review]

**Review by spookydonut — DISMISSED** [review]

**Comment by ExcessiveUseOfCobblestone** [comment]
I'm okay with flavortext still existing as an IC opener that kind of define what the author had for expectations of the role, but doesn't give explicit permission of actions that would have to be enforced by admins (IE you could have ash walker flavortext describe their tribal nature or how they know other off-lavaland lifeforms exist, but you shouldn't say that they can just kill indiscriminately in the flavortext).

Basically if it helps build the lore/story of the role, flavortext is fine. If it's meant to tell players how they can play the role, it MUST be policy'd.

File: .github/POLICYCONFIG.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+## Requirements
+It is a requirement of /tg/station development that all ghost roles, antags, minor antags and event mobs of any kind must support the policy system when implemented.
+
+## What is policy configuration
+Policy configuration is a json file that the administrators of a server can edit, which contains a dictionary of keywords -> string message.
+
+The policy text for a specific keyword should be displayed when relevant and appropriate, to allow server administrators to define the broad strokes of policy for some feature or mob.
+
+It is okay to provide a default text when the config is not set, but you are required to provide one in all cases of a ghost role or an antagonist, or minor event.
```
Comment (Rohesie) [inline .github/POLICYCONFIG.md]:
```suggestion
It is okay to provide a default text when the config is not set, but you are required to provide the config in all cases of a ghost role or an antagonist, or minor event.
```

**Comment by Rohesie** [comment]
Looks sensible.

**Review by Rohesie — DISMISSED** [review]

File: .github/POLICYCONFIG.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+It is a requirement of /tg/station development that all ghost roles, antags, minor antags and event mobs of any kind must support the policy system when implemented.
+
+## What is policy configuration
+Policy configuration is a json file that the administrators of a server can edit, which contains a dictionary of keywords -> string message.
+
+The policy text for a specific keyword should be displayed when relevant and appropriate, to allow server administrators to define the broad strokes of policy for some feature or mob.
+
+It is okay to provide a default text when the config is not set, but you are required to provide the config in all cases of a ghost role or an antagonist, or minor event.
+
+If you're in doubt about needing to support policy config, I suggest doing it anyway. This should replace all flavour text, ghost spawn messages and so forth that the player (i.e client) sees upon entering the role, mob, feature.
```
Comment (ExcessiveUseOfCobblestone) [inline .github/POLICYCONFIG.md]:
```suggestion
If you're in doubt about needing to support policy config, I suggest doing it anyway. This should replace all flavour text, ghost spawn messages and so forth that the player (i.e client) sees upon entering the role, mob, feature that is meant to dictate how they are permitted to be played as/with.
```
https://github.com/tgstation/tgstation/pull/50185#issuecomment-603876333

Comment (optimumtact) [reply .github/POLICYCONFIG.md]: that's not gramatically correct now dip ass, it should say role, mob or feature

Comment (ExcessiveUseOfCobblestone) [reply .github/POLICYCONFIG.md:15]: sorry english is my first language!

**Review by ExcessiveUseOfCobblestone — APPROVED** [review]

**Comment by ExcessiveUseOfCobblestone** [comment]
Since no one responded to my post 8 days ago i'm going to assume no one cares (but me) either way on the flavor text bit.
